### DISCUSSION TITLE
Customized android application is not compatible with some phones?

### DISCUSSION THREAD

**Discussion by pvasilek** [rustdesk/rustdesk-server-pro#447] [General]
Hello,

I am trying to install APK client generated with "Custom Clients" on Samsung A13 with Android 12 and get an error message during installation that application is not compatible with the current phone. But it works perfect with Samsung A23 and Samsung A32.
What special could be with A13 which makes it incompatible with the rustdesk application?

**Comment by 21pages** [comment]
Thanks for your feedback, it’s possible that the Samsung A13 has specific compatibility issues, but it's hard to pinpoint what makes it special.

**Comment by pvasilek** [comment]
What sort of compatibility issues it could be? Some permissions? Features? What are RustDesk Android requirements? Per Help page it is Android 6+/10+ only. Nothing else specified.

**Reply by MikeT-KBU** [reply]
Play Store installed on all 3 devices? If it isn't installed on A13 try installing it.

**Reply by pvasilek** [reply]
Yes, all devices are in the default configurations (non-rooted etc).
I suspect it may be incompatibility with min/target-sdk version specified in manifest + requested features and last Android OS updates.

At least I have a similar issue with another old APK which was solved by sdk retargeting and recompilation.
